drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/utils/ScenarioSimulationGridHeaderUtilities.java

```diff
@@ -90,10 +90,10 @@ public static ScenarioHeaderMetaData getColumnScenarioHeaderMetaData(final Scena
         return ci.getColumn();
     }
 
-    public static boolean isEditableHeader(final GridColumn<?> column,
+    public static boolean isEditableHeader(final ScenarioGridColumn column,
                                            final Integer uiHeaderRowIndex) {
         GridColumn.HeaderMetaData headerMetaData = column.getHeaderMetaData().get(uiHeaderRowIndex);
-        return headerMetaData instanceof ScenarioHeaderMetaData && !((ScenarioHeaderMetaData) headerMetaData).isReadOnly();
+        return headerMetaData instanceof ScenarioHeaderMetaData && !((ScenarioHeaderMetaData) headerMetaData).isReadOnly() && column.isEditableHeaders();
```

Comment:
Please add/update a test case to cover the fix

Reply:
Tests added :)